doc/changelog.md

```diff
@@ -236,33 +236,39 @@
   8.0
   ```
 
-  Either function can also be used independently of each other.
+  Either function can also be used independently of each other. Note that
+  `get_compilation_stage` replaces the `print_compilation_stage` function;
+  please see the Breaking Changes section for more details.
 
-* Catalyst now supports c executable generation with `catalyst.debug.compile_executable`.
-  A bug is fixed in `catalyst.debug.get_cmain` to support multi-dimensional arrays as
-  function inputs. 
+* Catalyst now supports direct C executable generation with `catalyst.debug.compile_executable`.
```

Comment:
This is very minor, but I don't know that `C executable` is too relevant because an executable is "written" in machine code, the fact that we generated it using the C language might not be too relevant? 😅

Reply:
Thank you, I wanted to rewrite this but wasn't sure what the best wording was! anything in mind?

Reply:
Something like this? :) 
```suggestion
* Catalyst now supports generating executables from compiled functions for the native host architecture using
  `catalyst.debug.compile_executable`.
```

Reply:
Perfect